docs/development/workflow/get_devel_version.rst

```diff
@@ -222,16 +222,14 @@ of `Astropy`_, for now this straightforward way will work (if you want to jump
 ahead to the more sophisticated method look at :ref:`virtual_envs`).
 
 .. note::
-    There are a couple of circumstances in which this quick method of
-    activating your copy of `Astropy`_ will NOT work and you need to go
-    straight to using a virtual python environment:
+    This quick method of activating your copy of `Astropy`_ will NOT work  
+    if you want to work on C or Cython code in `Astropy`_. 
+    You need to go straight to using a virtual python environment.
 
-    + You use Python 3.
-    + You want to work on C or Cython code in `Astropy`_.
 
 In the directory where your copy of `Astropy`_ is type::
 
-    python setup.py develop
+    python3 setup.py develop
```

Comment:
leave it as python, we don't support python 2 any more, thus python is by definition is assumed to be python3, etc.

Reply:
@bsipocz - I rather disagree - for many distributions this means you'll run `python2` - it is only when you activate a virtual environment that is python3 based that `python` equals `python3`. I would say that explicit is better than implicit here.

Reply:
:-1: on `python3` from me. What if when `python4` is available? This document would ask everyone to only install Astropy on Python 3 but not 4. Is that desirable? Wouldn't it be safer to have `setup.py` check for `sys.version` and raise an exception there for Python < 3?

Reply:
@mhvk - This is the dev docs for a python 3 only project. Is it too much to ask for prospective contributors to know whether they run python 2 or python 3 when they type `python`?

Reply:
I think it is silly for copy & paste not to give the right answer. Please realize that not everybody works on a mac with conda! On linux/debian, this simply fails. I had students trying to set up the development version today, and this failed for this reason (@fleaf5 is one of them). While if we change it to python3 we show that we're python3 only and it will always work (no python4 on the horizon; we can change that when that comes - after my retirement!).

p.s. The larger issue is that this text is rather more completely out of date, which is the problem with developers writing introductory text.

Reply:
Unsolicited opinion: why not drop the `python*` entirely? Then if we fix the shebang line in `setup.py` to use `python3` instead of `python`, no users will have to know and/or care. It will just do the right thing. And we'll also be in conformance with [PEP 394](https://www.python.org/dev/peps/pep-0394/).

Reply:
I like that - though it does become shell-specific whether `.` is in `PATH` - e.g., I would have to do `./setup.py` - which would not work in windows. I think that is an easier to parse problem than the wrong python interpreter though.

Reply:
If I see it correctly, the instructions in this page havewith the assumption that virtualenv is set up properly. I'm all for updating the page to recommend conda instead, but either way it doesn't seem to be an overstretched assumption that the user is in full control of the version numbers being used.
Btw on mac on cannot even use the system python, so it works even less out-of-the-box than the linux distros.

Anyway,  bottom line, if your experience is that we need python3 here, the let's have it, but if you could get anyone reworking the instructions to recommend conda (as it wouldn't hurt to get full control on a linux system either, and conda works the same way on all platforms) based on the fresh experience, then it's even better.

Reply:
For `develop`, as I read it and as I saw the students execute it, the instructions assume one has *not* set up virtual environments - that is the next step. Indeed, the whole point of `setup.py develop` is to avoid having to go to a virtual environment.

Reply:
What about recommending ``pip install -e .`` instead?

Reply:
Note also that ``python3`` does work on MacOS X with conda.

Reply:
But we could just mention both ``python`` and ``python3?``?

Reply:
I like the `pip install -e` workaround.

Reply:
Let's just hope it's not ``pip3`` on some platforms ;)

Reply:
well, then we go back to my first point above, that some assumption about basic knowledge and control of the python environment is not really that unreasonable for the prospective contributors.